src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php

```diff
@@ -1025,6 +1027,42 @@ protected function createChildContext(array $parentContext, string $attribute, ?
 
         $this->assertFalse($normalizer->childContextCacheKey);
     }
+
+    public function testNormalizationWithMaxDepthOnStdclassObjectDoesNotThrowWarning()
+    {
+        $object = new \stdClass();
+        $object->string = 'yes';
+
+        $loader = new LoaderChain([
+            new AttributeLoader(),
+        ]);
+        $classMetadataFactory = new ClassMetadataFactory($loader);
+
+        $serializer = new Serializer([
+            new ObjectNormalizer($classMetadataFactory),
+        ], [
+            new JsonEncoder(),
+        ]);
+
+        $page = [
+            'props' => [
+                'property' => [
+                    'object' => $object,
+                ],
+            ],
+        ];
+
+        $context = [];
+
+        $json = $serializer->serialize($page, 'json', array_merge([
```

Comment:
As context is empty, this `array_merge` is not needed

Reply:
removed